frontend/packages/system/src/NotificationEvents/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect } from 'react';
+import { useBreadcrumbs, useQueryParamsState } from '@common/hooks';
+import {
+  AppBarButtonsPortal,
+  AppBarContentPortal,
+  BasicSpinner,
+  Box,
+  EditIcon,
+  LoadingButton,
+  RelativeTimeDate,
+  Stack,
+  TextArea,
+  Typography,
+} from '@common/ui';
+import { useTranslation } from '@common/intl';
+import { useNotificationEvents } from '../api';
+import {
+  ConfigKind,
+  EventStatus,
+  useNavigate,
+  useParams,
+} from 'packages/common/src';
+import { configRoute } from '../../Notifications/navigate';
+import { NotificationStatusChip } from '../components/NotificationStatusChip';
+
+export const DetailView = () => {
+  const t = useTranslation('system');
+  const urlParams = useParams();
+  const { suffix, setSuffix } = useBreadcrumbs();
+  const navigate = useNavigate();
+
+  const { queryParams } = useQueryParamsState({
+    initialFilter: { id: { equalTo: urlParams['id'] } },
+  });
+
+  const { data, isLoading } = useNotificationEvents(queryParams);
+  const entity = data?.nodes[0];
+
+  useEffect(() => {
+    const listName = entity?.title;
+    if (!suffix && listName) {
+      setSuffix(listName);
+    }
+  }, [suffix, entity]);
+
+  return (
+    <>
+      <AppBarButtonsPortal>
+        {/* if we have a config_id, create a link to edit the config */}
+        {entity?.notificationConfigId && (
+          <LoadingButton
+            isLoading={isLoading}
+            onClick={() => {
+              navigate(
+                configRoute(
+                  entity.notificationConfig?.kind ?? ConfigKind.Scheduled,
+                  entity.notificationConfigId ?? ''
+                )
+              );
+            }}
+            variant="outlined"
+            endIcon={<EditIcon />}
+          >
+            {t('button.edit-config')} ({entity.notificationConfig?.title})
+          </LoadingButton>
```

Comment:
this doesn't need to be a LoadingButton - it's only navigating, so there's no loading state to display

```suggestion
          <BaseButton
            onClick={() => {
              navigate(
                configRoute(
                  entity.notificationConfig?.kind ?? ConfigKind.Scheduled,
                  entity.notificationConfigId ?? ''
                )
              );
            }}
            variant="outlined"
            endIcon={<EditIcon />}
          >
            {t('button.edit-config')} ({entity.notificationConfig?.title})
          </BaseButton>
```